srcpkgs/SDL2/template

```diff
@@ -15,7 +15,7 @@ license="Zlib"
 homepage="https://www.libsdl.org/"
 changelog="https://raw.githubusercontent.com/libsdl-org/SDL/main/WhatsNew.txt"
 distfiles="https://www.libsdl.org/release/${pkgname}-${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://www.libsdl.org/release/SDL2-${version}.tar.gz"
```

Reply:
Is there a change about using the ${pkgname} on the URLs?